constraints.txt

```diff
@@ -1 +1,2 @@
 cftime<1.5
+compliance-checker==5.4.2
```

Comment:
`compliance-checker` is a requirement, not a constraint. It's currently specified in the [setup.py](https://github.com/aodn/cc-plugin-imos/blob/c449e252ad790ff734aca25b0f1c184becf9d206/setup.py#L15), so let's set a _minimum version_ there (if you can find out which version the breaking change was made in)